example/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ if(RANGES_BUILD_CALENDAR_EXAMPLE)
     set(Boost_USE_STATIC_LIBS        ON)
     set(Boost_USE_MULTITHREADED     OFF)
     set(Boost_USE_STATIC_RUNTIME    OFF)
-    find_package(Boost 1.59 COMPONENTS date_time program_options)
+    find_package(Boost 1.59.0 COMPONENTS date_time program_options)
```

Comment:
Should this version number be bumped to 1.63.0, since that's what travis is going to download and build?